.github/workflows/flatpak-update.yml

```diff
@@ -0,0 +1,41 @@
+name: Update Flatpak YAML to use latest version of ASIMOV CLI
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 * * 4"
+
+jobs:
+  updateFlatpak:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+
+      - name: Set up Git user
+        run: |
+          git config user.name "github-actions[bot]"
+          git config user.email "github-actions[bot]@users.noreply.github.com"
+
+      - name: Update Flatpak YAML
+        shell: bash
+        run: |
+          set -euxo pipefail
+          
+          FLATPAK_YAML="flatpak/com.asimov_platform.asimov_cli.yaml"
+          LATEST_VERSION="$(curl -s https://api.github.com/repos/asimov-platform/asimov-cli/releases/latest | jq -r .tag_name)"
+          
+          sed -i "s|https://github.com/asimov-platform/asimov-cli/releases/download/[^/]*|https://github.com/asimov-platform/asimov-cli/releases/download/${LATEST_VERSION}|g" "$FLATPAK_YAML"
+          
+          x86_64_hex="$(curl -sL "https://github.com/asimov-platform/asimov-cli/releases/download/${LATEST_VERSION}/asimov-linux-x86-gnu.gz.sha256" | cut -d ' ' -f1)"
+          sed -i -E "/asimov-linux-x86-gnu.gz/{N;s/(sha256: \").*(\".*)/\1${x86_64_hex}\2/}" "$FLATPAK_YAML"
+          
+          aarch64_hex="$(curl -sL "https://github.com/asimov-platform/asimov-cli/releases/download/${LATEST_VERSION}/asimov-linux-arm-gnu.gz.sha256" | cut -d ' ' -f1)"
+          sed -i -E "/asimov-linux-arm-gnu.gz/{N;s/(sha256: \").*(\".*)/\1${aarch64_hex}\2/}" "$FLATPAK_YAML"
+          
+          if ! git diff --exit-code "$FLATPAK_YAML"; then
+            git add "$FLATPAK_YAML"
+            git commit -m "chore: Update Flatpak YAML to ${LATEST_VERSION}"
```

Comment:
We're not using conventional commits, so no need for the "chore:" prefix here.

Reply:
@artob it is done in - https://github.com/asimov-platform/asimov-packaging/commit/f32116ad765ef6328d08e3916194ad8617e0be99